src/overlays/ovl_i6/fox_ve2.c

```diff
@@ -0,0 +1,4132 @@
+                    D_80177988 += D_80177A48[4];
+                    Math_SmoothStepToF(&D_80177A48[4], 20.0f, 1.0f, 0.1f, 0.0f);
+                } else {
+                    D_80177988 = gActors[10].obj.pos.z - 20.0f;
+                }
+                if ((D_80177A80 >= 920) && (gTeamShields[3] > 0)) {
+                    if (D_80177A80 == 920) {
+                        player->timer_1F8 = 25;
+                    }
+                    gActors[10].fwork[19] = __sinf((f32) player->timer_1F8 * 40.0f * M_DTOR) * 5.0f;
+                } else {
+                    if (D_80177A80 < 850) {
+                        switch (D_80177A80) {
+                            case 600:
+                                D_80177A48[2] = 80.0f;
+                                D_80177A48[3] = -45.0f;
+                                break;
+                            case 650:
+                                D_80177A48[2] = 0.0f;
+                                break;
+                            case 680:
+                                D_80177A48[2] = 45.0f;
+                                D_80177A48[3] = -20.0f;
+                                break;
+                            case 750:
+                                D_80177A48[2] = 90.0f;
+                                D_80177A48[3] = -30.0f;
+                                break;
+                            case 780:
+                                D_80177A48[2] = 30.0f;
+                                D_80177A48[3] = -45.0f;
+                                break;
+                            case 830:
+                                D_80177A48[2] = -20.0f;
+                                D_80177A48[3] = -25.0f;
+                                break;
+                            default:
+                                // D_80177A48[2] = 0.0f;
+                                break;
+                        }
+                    }
+                    if (D_80177A80 > 880) {
+                        Math_SmoothStepToF(&gActors[10].fwork[19], 0.0f, 0.2f, 2.0f, 0.0f);
+                        Math_SmoothStepToF(&gActors[10].fwork[20], 0.0f, 0.2f, 2.0f, 0.0f);
+                    } else {
+                        Math_SmoothStepToF(&gActors[10].fwork[19], D_80177A48[2], 0.2f, 2.0f, 0.0f);
+                        Math_SmoothStepToF(&gActors[10].fwork[20], D_80177A48[3], 0.2f, 2.0f, 0.0f);
+                    }
+                }
+                D_80177A48[0] = 0.01f;
+                Math_SmoothStepToF(&D_80177A48[1], 10.0f, 1.0f, 0.1f, 0.0f);
+                Math_SmoothStepToF(&player->camEye.x, gActors[10].obj.pos.x - 50.0f, D_80177A48[0], D_80177A48[1], 0);
+                Math_SmoothStepToF(&player->camEye.y, gActors[10].obj.pos.y + 10.0f, D_80177A48[0], D_80177A48[1], 0);
+                Math_SmoothStepToF(&player->camEye.z, D_80177988, D_80177A48[0], D_80177A48[1], 0);
+                D_801779B8 = __sinf((f32) gGameFrameCount * 5.5f * M_DTOR) + 15000.0f;
```

Comment:
```suggestion
                D_801779B8 = __sinf(gGameFrameCount * 5.5f * M_DTOR) + 15000.0f;
```
It should be fine to leave these conversions implicit as long as it still matches.